.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,13 @@
 <!--- Make sure to add a descriptive title in the field above! E.g. "Fixes the header title color on the homepage"  -->
 
+## Ember Times review template
+
+- [ ] Author's name at top and bottom
```

Comment:
I think the word, author, may be confusing since we tend to cover articles written by other people.

```suggestion
- [ ] Add your name to top and bottom
```

Reply:
Agree